spaces/views.py

```diff
@@ -51,6 +53,30 @@ def get(self, request):
         ]
         return JsonResponse({"space_card":self.space_card, "review_card":review_card}, status = 200)
 
+class LikeView(View):
+    @login_required
+    def patch(self, request, space_id):
+        try:
+            space    = Space.objects.get(id = space_id)
+            user     = request.user
+            like     = Like.objects.get(user = user, space = space)
+
+            if like.is_liked == True:
+                like.is_liked = False
+                like.save()
+                return JsonResponse({"message":"UNLIKE"}, status = 200)
+
+            like.is_liked = True
+            like.save()
+            return JsonResponse({"message":"LIKE"}, status = 200)
+        except Like.DoesNotExist:
+            Like.objects.create(user = user, space = space, is_liked = True)
```

Comment:
```suggestion
          if Like.objects.filter(user = user, space = space).exists():
              like     = Like.objects.get(user = user, space = space)
              if like.is_liked == True:
                  like.is_liked = False
                  like.save()
                  return JsonResponse({"message":"UNLIKE"}, status = 200)
              like.is_liked = True
              like.save()
              return JsonResponse({"message":"LIKE"}, status = 200)
          Like.objects.create(user = user, space = space, is_liked = True)
```
Exception으로 처리한다는것은 에러를 의미하는데, 
거기서 라이크를 생성하는건 부자연스럽습니다.  
위와 같이 조건 처리 하는게 더 보기 좋습니다.

Reply:
알겠습니다 😁
다른 방법도 조금만 더 생각해보고 푸시 하겠습니다